dll/win32/ieframe/lang/ro-RO.rc

```diff
@@ -66,24 +66,24 @@ STRINGTABLE
 
 STRINGTABLE
 {
-    IDS_FINDINGRESOURCE     "În căutarea resursei %s"
-    IDS_BEGINDOWNLOADDATA   "În început de descărcare din %s"
-    IDS_ENDDOWNLOADDATA     "În curs de descărcare din %s"
-    IDS_SENDINGREQUEST      "În așteptarea răspunsului de la %s"
+    IDS_FINDINGRESOURCE     "Se caută %s"
+    IDS_BEGINDOWNLOADDATA   "Începe descărcarea %s"
+    IDS_ENDDOWNLOADDATA     "Se descarcă %s"
+    IDS_SENDINGREQUEST      "Se interoghează %s"
 }
 
 
 IDD_BROWSE_OPEN DIALOG 10, 10, 200, 70
 STYLE DS_MODALFRAME | WS_CAPTION
-CAPTION "Deschide URL"
+CAPTION "Deschidere URL"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "Specificați adresa URL pe care doriți s-o deschideți în Internet Explorer",-1, 35,5,160,25
-    LTEXT "Deschide:", -1, 5, 32, 30, 15
+    LTEXT "Specificați adresa URL pe care doriți să o deschideți în Internet Explorer",-1, 35,5,160,25
+    LTEXT "Deschidere:", -1, 5, 32, 30, 15
```

Comment:
Prove with a pic that the longer string "Deschidere:" would actually fit the 30 width. I do have my doubt on that.

Reply:
I will revert this change. It doesn't fit in the box. 

![2024-09-25 21_22_15-Dialog - 1001](https://github.com/user-attachments/assets/bc8082f9-c609-4390-b31f-bd7907a0f53c)